test/check_sharding_metadata_test.lua

```diff
@@ -527,3 +527,25 @@ function g.test_sharding_func_invalid_name_component()
     -- remove test data
     rawset(_G, user_sharding_func_name, nil)
 end
+
+
+local vshard_cases = {
+    mpcrc32 = {
+        sharding_func_name = 'vshard.router.bucket_id_mpcrc32',
+    },
+    strcrc32 = {
+        sharding_func_name = 'vshard.router.bucket_id_strcrc32',
+    }
+}
+
+for name, case in pairs(vshard_cases) do
+    local test_name = ('test_vshard_%s_sharding_func'):format(name)
+
+    g[test_name] = function(g)
+        g.space.sharding_func = case.sharding_func_name
+
+        local res, err = ddl.check_schema(g.schema)
+        t.assert_equals(err, nil)
+        t.assert_equals(res, true)
+    end
+end
```

Comment:
Well, such approach is more human-readable.

Reply:
So, what parameterization approach do we choose?

Reply:
I'm not a big fan of luatest parametrization, so it's up to you from my side. Both approaches should be ok here.

Reply:
I decided to use luatest's parameterization in `ddl` module since there is only one parameter and we haven't used custom parameterization in `ddl` before. But it is reasonable for `crud` to have custom parameterization in tests.